hw/ip/rv_dm/rtl/deps.f

```diff
@@ -0,0 +1,21 @@
+${PRJ_DIR}/hw/top_earlgrey/rtl/top_pkg.sv
```

Comment:
Seems like those files got re-introduced as part of the rebase.

Reply:
uh... looks like I messed up the rebase.